### PR TITLE
Update the Lurker library dependency to ~1.1@dev

### PR DESCRIPTION
**Pull request by szymach**
The Lurker library in version ~1.0 has an issue, where sometimes it is impossible to watch directories. I propose updating it to version ~1.1 where I did not encounter this problem.

### CODE REVIEW

**Comment by jjok** [comment]
:+1: I've been investigating this issue all afternoon. Just worked out where the issue is.

**Comment by jjok** [comment]
Do we know why Robo uses a fork of Lurker? flint/Lurker seems to be more up to date. It has a 1.1 tag, whereas henrikbjorn/Lurker doesn't.

**Comment by szymach** [comment]
Don't really know, I had to fork the Robo lib and use it in my project to make things work.

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "symfony/console": "~2.5",
         "symfony/process": "~2.5",
         "symfony/filesystem": "~2.5",
-        "henrikbjorn/lurker": "1.0.*@dev"
+        "henrikbjorn/lurker": "~1.1@dev"
```
Comment (jjok) [inline composer.json]: I would change it to `"flint/lurker": "~1.0"`. That should get you the 1.1 branch, and I think that version is more actively developed.

**Comment by DavertMik** [comment]
> Do we know why Robo uses a fork of Lurker? flint/Lurker seems to be more up to date. It has a 1.1 tag, whereas henrikbjorn/Lurker doesn't.

In the days it was initially born, this version was the most stable one. Not sure, maybe its time to switch to other.

File: composer.json (hunk 1 above)
Comment (DavertMik) [inline composer.json]: There is no flint/lurker library on packagist:
```
https://github.com/flint/Lurker/blob/master/composer.json#L2
```

Comment (szymach) [inline composer.json]: Right, it seems the repository is flint/lurker, but on packagist it is henrikbjorn/lurker. https://github.com/flint/Lurker So it is not a fork, simply the dependency version needs to be updated.

**Comment by DavertMik** [comment]
Done. Also updated symfony version to match 3.x as well

**Pull request closed without merging**